README.md

```diff
@@ -112,15 +112,13 @@ RQ: Do the certainty scores in CLIP3 predictions correspond to the actual catego
 ## Methods, results
 To answer the above, I examined the 'first-choice category' values in the primary data, focussing mainly on the highest and the lowest confidence values to scan the data for possible patterns. To interpret any tendencies found, I used preliminary evaluation data produced during the Hackathon ('Secondary data' above), and close-reading of the images themselves. 
 
-To test H4a/b and H5a/b, I started by filtering out entries with the highest and the lowest first-choice confidence scores in the following ranges:<br>
+To test H4a/b and H5a/b, I started by filtering out entries with the highest and the lowest first-choice confidence scores in the following ranges:
+
 **'Low-confidence' subset**  (2734 entries)<br>
       x ∈ [0;30) <br>
-i.e.  0 ≤ x < 30.(0)
```

Comment:
I think one of these is clear enough – I'd prefer `0.0 ≤ x < 30.0`, but I left the format you use in the tables